recipes/api.rb

```diff
@@ -64,79 +138,74 @@
 yum_package 'postgresql93-devel'
 yum_package 'postgresql93-contrib'
 
+log 'Install gem pg'
 bash 'gem instal pg' do
-  cwd '/opt/'
-  user 'root'
+  cwd '/home/jellyfish/'
+  user 'jellyfish'
   code <<-EOH
-  source /etc/profile.d/rbenv.sh
-  rbenv global 2.1.5
-  /opt/rbenv/versions/2.1.5/bin/gem install pg -v '0.17.1' -- \
-  --with-pg-config=/usr/pgsql-9.3/bin/pg_config
+   source /home/jellyfish/.bash_profile
+   /home/jellyfish/.rbenv/bin/rbenv global 2.2.0
```

Comment:
These lines should use default['rbenv']['user_home'] not hard coded "/home/jellyfish"